app/Http/Controllers/ZhanrController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\Zhanr\CreateRequest;
+use App\Http\Requests\Zhanr\EditRequest;
+use App\Models\Zhanr;
+use Illuminate\Http\Request;
+
+class ZhanrController extends Controller
+{
+
+    public function createForm()
+    {
+        return view('zhanrs.create');
+    }
+
+    public function editForm(int $id)
+    {
+        $zhanr = Zhanr::query()->findOrFail($id);
```

Comment:
давай тоже через роут биндинг 
```suggestion
    public function editForm(Zhanr $zhanr)
    {
```